### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by dmbch**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "postcss"
   ],
   "scripts": {
-    "start": "cd demo; rm -rf dist node_modules/*hops*; yarn install --force; yarn start",
+    "start": "cd demo; rm -rf dist node_modules/*hops*; npm install --no-package-lock; npm start",
```
Comment (robin-drexler) [inline package.json]: Why `--no-package-lock`? Asking because  there's been no `--no-lockfile` when yarn was used?

Comment (dmbch) [reply package.json]: I probably forgot to add it in the yarn case. Generally, I am trying here to force the package manager to fetch the freshest versions of all packages to simulate the behaviour in Travis (and in actual new projects).

**Comment by dmbch** [comment]
@robin-drexler PTAnotherL :)

**Comment by robin-drexler** [comment]
ja

**Pull request merged**